core/src/com/po/fuck/view/FollowingDrawer.java

```diff
@@ -0,0 +1,33 @@
+package com.po.fuck.view;
+
+import static com.po.fuck.view.Constants.GAME_HEIGHT;
+import static com.po.fuck.view.Constants.GAME_WIDTH;
+
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.ObjectFollower;
+
+public class FollowingDrawer extends CenterDrawer {
+
+    private ObjectFollower objectFollower;
+
+    public FollowingDrawer(SpriteBatch batch, ObjectFollower objectFollowing){
+        super(batch);
+        this.objectFollower = objectFollowing;
+    }
+
+    @Override
+    public void draw(Sprite sprite, Vector2 position) {
+        super.draw(sprite, position.cpy()
+            // Subtract the position of the object being followed from the copied position
+            .sub(this.objectFollower.getPosition())
+            // Add half the game width and height to center the sprite on the screen
+            .add(GAME_WIDTH / 2, GAME_HEIGHT / 2));
```

Comment:
I know how to read, but can you explain why is this formula correct/why are you doing this ?

Reply:
Ask yourself? https://github.com/MAKMED1337/PO/blob/de7549fac28494e3202efa52931f46ca64749874/core/src/com/po/fuck/Camera.java#L42

Reply:
Here are you go https://github.com/MAKMED1337/PO/commit/84babb630335c54d0127aad6ff5dce0cf4811919